### ISSUE TITLE
[CoE Starter Kit - BUG] ISSUE TITLE

### ISSUE THREAD

**Issue by akshaya-ak** [microsoft/coe-starter-kit#5673]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Table Name: Connection Reference Identity

Connection: RSS and Approvals are not visible in the above-mentioned table.
However, we tried adding it Manually, but it is getting cleared. 

### Expected Behavior

All connections should be visible in the table for that particular environment.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.8

### What app or flow are you having the issue with?

Approvals and RSS

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by akshaya-ak** [comment]
Similarly- 
1) SharePoint
2) Power query
3) Data flows 

are also not visible in the PBI reports for other environments.

**Comment by Jenefer-Monroe** [comment]
I see everything from your list working aside from Approvals and RSS. 
Neither Approvals nor RSS have an identity associated with them which you can see by creating a new one, neither has you sign in. 

![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/69ea8d5a-067b-49c8-8bb1-995994430eeb)

![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/07c2bde9-f225-4c01-b18a-ed45a927e07e)

As a result we did not add them here. A bunch of the logic in the flow assumes an identity will be included. Hope that makes sense.

**Comment by akshaya-ak** [comment]
Hi Jenefer, Thank you for your quick response!

These 2 connections are already added and connected.
![image](https://github.com/microsoft/coe-starter-kit/assets/71701878/d3819048-b382-479d-bd0e-8b2e628352e9)

**Comment by Jenefer-Monroe** [comment]
Correct. What I mean is that they are not connected via an identity. As you can se when you created these two vs the others in the list, or when they are displayed here. They do not list your name next to them.

**Comment by akshaya-ak** [comment]
so, this is the expected behavior? the table will not have RSS and Approvals connector.

**Comment by akshaya-ak** [comment]
okay thank you for your confirmation.
